webapp/src/components/App/Studies/LauncherDialog.tsx

```diff
@@ -53,8 +54,9 @@ function LauncherDialog(props: Props) {
   const { enqueueSnackbar } = useSnackbar();
   const enqueueErrorSnackbar = useEnqueueErrorSnackbar();
   const [options, setOptions] = useState<LaunchOptions>({
-    nb_cpu: LAUNCH_LOAD_DEFAULT,
+    nb_cpu: DEFAULT_NB_CPU,
     auto_unzip: true,
+    time_limit: DEFAULT_TIME_LIMIT,
```

Comment:
We should do it for all other options or we'll have the same issue some day with those options.
In the type `LaunchOptions`, nothing should be optional, to ensure we always send all displayed values.

Reply:
Thank you for your feedback. If I'm right the backend already assigns default values to these options in the `LauncherParametersDTO` class, which mitigates potential issues. 

The one missing were `nb_cpu` and `time_limit` that I added, and according to @laurent-laporte-pro  the `nb_cpu` can be tricky to handle as default because it's not the same for all environments, I need some more precisions about that.

Also please note that we're dealing with legacy code, and many issues comes from that as outlined in this issue: 
[ Rework LauncherDialog for Improved User and Technical Experience #1783 ](https://github.com/AntaresSimulatorTeam/AntaREST/issues/1783)

When the form will be refactored, all values will be validated against constraints(min,max, valid input etc) and default values will be explicitly set in the inital state of the from.

I'm leaning towards not duplicating defaults in the frontend, considering the backend already handles them and our current legacy code scenario. But I'm open to other perspectives. What’s your take? Looking forward on this.